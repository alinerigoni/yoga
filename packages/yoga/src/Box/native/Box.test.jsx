import React from 'react';
import { render } from '@testing-library/react-native';

import { ThemeProvider, Box, Text } from '../..';

describe('<Box />', () => {
  it('should match snapshot', () => {
    const { container } = render(
      <ThemeProvider>
        <Box
          b="small"
          p="small"
          borderRadius="small"
          color="feedback.success.dark"
          as={Text}
        >
          Making wellbeing universal
        </Box>
      </ThemeProvider>,
    );

    expect(container).toMatchSnapshot();
  });
});
