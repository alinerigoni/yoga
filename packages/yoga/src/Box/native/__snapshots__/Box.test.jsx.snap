// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Box /> should match snapshot 1`] = `
<View
  collapsable={true}
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <Text
    b="small"
    borderRadius="small"
    color="feedback.success.dark"
    inverted={false}
    light={false}
    p="small"
    size="medium"
    style={
      Array [
        Object {
          "color": "#231B22",
          "fontFamily": "Rubik",
          "fontSize": 16,
          "fontWeight": "400",
          "lineHeight": 24,
        },
        Object {
          "borderColor": "black",
          "borderRadius": 8,
          "borderStyle": "solid",
          "borderWidth": 1,
          "color": "#1D856C",
          "paddingBottom": 16,
          "paddingLeft": 16,
          "paddingRight": 16,
          "paddingTop": 16,
        },
      ]
    }
  >
    Making wellbeing universal
  </Text>
</View>
`;
